utils/order-utils/test/orders.js

```diff
@@ -146,7 +146,7 @@ describe('Orders', async () => {
       GANACHE_PROVIDER
     )
 
-    order.signature.v += 1
+    order.signature.v -= 1
```

Comment:
This was added since the order.signature.v is now 28 and adding one would take it out of the range, so instead subtracted 1.

Reply:
Nice catch